framework/src/org/apache/cordova/engine/SystemWebChromeClient.java

```diff
@@ -176,7 +176,6 @@ public void onGeolocationPermissionsShowPrompt(String origin, Callback callback)
         {
             geolocation.requestPermissions(0);
         }
-
```

Comment:
ditto on cleanup, message like "other minor cleanup" in description and commit messages would be nice

Reply:
Extracted the whitespace changes to a separate commit and updated the PR description.